delta/sourcing-psql/src/main/scala/ch/epfl/bluebrain/nexus/delta/sourcing/FragmentEncoder.scala

```diff
@@ -0,0 +1,21 @@
+package ch.epfl.bluebrain.nexus.delta.sourcing
+
+import doobie.util.fragment.Fragment
+
+/**
+  * A type class that provides a conversion from a value of type `A` to a doobie [[Fragment]].
+  */
+trait FragmentEncoder[A] {
```

Comment:
This case class alloew to convert a A to a doobie fragment in the same fashion as we do for circe encoders for example